src/Resources/contao/library/Contao/Widget.php

```diff
@@ -263,6 +265,14 @@ public function __set($strKey, $varValue)
 				$this->strPrefix = $varValue;
 				break;
 
+			case 'useRawRequestData':
```

Comment:
Since you have declared the property as a `boolean`, we should handle the `false` case as well, shouldn't we?